prisma/migrations/20250226072140_add_passkey/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateTable
```

Comment:
@matteyu Could you instead re-generate the `init` migration. As we don't have a production DB yet, we can simply drop the DB and start with a fresh one. I'd like to only have migrations for triggers or changes we do without prisma until we release the production version.

Reply:
@matteyu For regenerating we do have a script as well https://github.com/wanderwallet/embed-api/blob/development/scripts/regenerate-prisma-migrations.sh which you can update in this PR to include the necessary migration folders and run using `pnpm db:regenerate-migrations`